### ISSUE TITLE
Uncaught Error: ViewPropTypes has been removed from React Native

### ISSUE THREAD

**Issue by danjayrose** [GetStream/react-native-activity-feed#295]
- iOS or Android? Both
- Expo or regular React Native? React Native
- React Native Version: 0.69.4
- react-native-activity-feed@1.1.1
- getstream@7.2.11

**Describe the bug**
When installing with the latest React Native version this package causes an uncaught error as it uses ViewPropTypes which have been deprecated by React Native.

**To Reproduce**

1. Create new project with react native
2. Install react-native-activity-feed@1.1.1 and import in the code

**Expected behavior**
As per recommendation in https://github.com/facebook/react-native/issues/33557#issuecomment-1212212561 if the import is updated to use deprecated-react-native-prop-types it should work

**Comment by TianrenWang** [comment]
LOL was expecting to find this question at the top of the issue. If they fix this, great, but I don't expect them to do that any time soon, maybe ever.

**Comment by danjayrose** [comment]
> 

Yes I also thought the same as it seems that this isn't maintained anymore. To resolve the issue for my project I use patch-package to resolve the problem that seems to originate from react-native-keyboard-spacer.

**Comment by TianrenWang** [comment]
> > 
> 
> Yes I also thought the same as it seems that this isn't maintained anymore. To resolve the issue for my project I use patch-package to resolve the problem that seems to originate from react-native-keyboard-spacer.

@danjayrose By any chance, can I get a copy of that patch-package?

**Comment by danjayrose** [comment]
> 

Sorry for the late reply, I've been on holiday. I've attached the patch file, just rename the file extension from .txt to .patch and put it in your patches folder

[react-native-keyboard-spacer+0.4.1.txt](https://github.com/GetStream/react-native-activity-feed/files/9504530/react-native-keyboard-spacer%2B0.4.1.txt)
